presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -430,13 +432,13 @@ public int getSchedulingWeight()
     }
 
     @Override
-    public void setSchedulingWeight(int weight)
```

Comment:
Unnecessary change? `weight` seemed like a perfectly valid name for the parameter.

Reply:
Unintended, yes.